pkg/controller/migplan/validation.go

```diff
@@ -43,18 +46,20 @@ const (
 
 // Messages
 const (
-	ReadyMessage                        = "The migration plan is ready."
-	InvalidSourceClusterRefMessage      = "The `srcMigClusterRef` must reference a `migcluster`."
-	InvalidDestinationClusterRefMessage = "The `dstMigClusterRef` must reference a `migcluster`."
-	InvalidStorageRefMessage            = "The `migStorageRef` must reference a `migstorage`."
-	InvalidAssetCollectionRefMessage    = "The `migAssetCollectionRef` must reference a `migassetcollection`."
-	SourceClusterNotReadyMessage        = "The referenced `srcMigClusterRef` does not have a `Ready` condition."
-	DestinationClusterNotReadyMessage   = "The referenced `dstMigClusterRef` does not have a `Ready` condition."
-	StorageNotReadyMessage              = "The referenced `migStorageRef` does not have a `Ready` condition."
-	AssetCollectionNotReadyMessage      = "The referenced `migAssetCollectionRef` does not have a `Ready` condition."
-	AssetNamespaceNotFoundMessage       = "The following asset `namespaces` [%s] not found on the source cluster."
-	InvalidDestinationClusterMessage    = "The `srcMigClusterRef` and `dstMigClusterRef` cannot be the same."
-	EnsureStorageFailedMessage          = "Failed to create/validate backup and volume snapshot storage."
+	ReadyMessage                          = "The migration plan is ready."
+	InvalidSourceClusterRefMessage        = "The `srcMigClusterRef` must reference a `migcluster`."
+	InvalidDestinationClusterRefMessage   = "The `dstMigClusterRef` must reference a `migcluster`."
+	InvalidStorageRefMessage              = "The `migStorageRef` must reference a `migstorage`."
+	InvalidAssetCollectionRefMessage      = "The `migAssetCollectionRef` must reference a `migassetcollection`."
+	SourceClusterNotReadyMessage          = "The referenced `srcMigClusterRef` does not have a `Ready` condition."
+	DestinationClusterNotReadyMessage     = "The referenced `dstMigClusterRef` does not have a `Ready` condition."
+	StorageNotReadyMessage                = "The referenced `migStorageRef` does not have a `Ready` condition."
+	AssetCollectionNotReadyMessage        = "The referenced `migAssetCollectionRef` does not have a `Ready` condition."
+	AssetNamespaceNotFoundMessage         = "The following asset `namespaces` [%s] not found on the source cluster."
+	InvalidDestinationClusterMessage      = "The `srcMigClusterRef` and `dstMigClusterRef` cannot be the same."
+	EnsureStorageFailedMessage            = "Failed to create/validate backup and volume snapshot storage."
+	NsNotFoundOnSourceClusterMessage      = "The follow namespaces [%s] not found on source cluster."
```

Comment:
Assume this was meant to read:

`The following namespaces`

I think we could reword this to 

`Namespaces [%s] not found on source cluster`

Reply:
Fixed.